satpy/tests/reader_tests/test_fci_l1c_nc.py

```diff
@@ -599,6 +754,30 @@ def test_orbital_parameters_attr(self, reader_configs, fh_param):
                 "projection_altitude": 35786400.0,
             }
 
+    @pytest.mark.parametrize("channel",list_total_channel)
+    @pytest.mark.parametrize("resolution",list_resolution)
+    def test_orbital_parameters_attr_af(self,FakeFCIFileHandlerAF_fixture,reader_configs,channel):
+        """Test the orbital parametters for AF data."""
+        expected_res_n = 1
+        fh_param = FakeFCIFileHandlerAF_fixture
+        reader = _get_reader_with_filehandlers(fh_param["filenames"], reader_configs)
+        type_ter = self._get_type_ter_AF(channel)
+        res = reader.load([make_dataid(name=name)
+                for name in fh_param["channels"][type_ter]], pad_data=False)
+        assert expected_res_n == len(res)
+        for ch in fh_param["channels"][type_ter]:
+            assert res[ch].attrs["orbital_parameters"] == {
+                "satellite_actual_longitude": np.mean(np.arange(6000)),
+                "satellite_actual_latitude": np.mean(np.arange(6000)),
+                "satellite_actual_altitude": np.mean(np.arange(6000)),
+                "satellite_nominal_longitude": 0.0,
+                "satellite_nominal_latitude": 0,
+                "satellite_nominal_altitude": 35786400.0,
+                "projection_longitude": 0.0,
+                "projection_latitude": 0,
+                "projection_altitude": 35786400.0,
+            }
+
```

Comment:
since the AF filetype has the same structure and it uses exactly the same code for this part in the reader, this test is not necessary
```suggestion

```